cli/user/README.md

```diff
@@ -94,7 +95,52 @@ All available commands:
 	stream read scoped-stream-name [group-similar] [timeout-in-seconds]: Reads all Events from a Stream and then tails the Stream.
 ```
 
+You can try out commands such as:
+```
+> scope create example
+Scope 'example' created successfully.
+```
+
 For more info on how the Pravega User CLI works, please visit [this page](https://github.com/pravega/pravega/wiki/Pravega-User-CLI).
 
+## Pravega CLI on Kubernetes
+The pravega-cli can be accessed through the kubernetes pod such as the segmentstore pod.
```

Comment:
The Pravega User CLI is different, as it can be accessed externally if we expose the Controller/Segment Store endpoints to the outside world. So, here you need to add to what I suggested in the previous README (it can be used inside the cluster, either on a new pod or in one of the existing server instances) the option to be used from outside, as long as the Pravega cluster is externally accesible.